lib/SIL/SILFunctionBuilder.cpp

```diff
@@ -83,6 +83,14 @@ void SILFunctionBuilder::addFunctionAttributes(SILFunction *F,
       !constant.autoDiffDerivativeFunctionIdentifier &&
       !constant.isStoredPropertyInitializer() &&
       !constant.isThunk()) {
+    // NOTE: Validate `@differentiable` attributes on `AccessorDecl`s by calling
```

Comment:
I don't completely understand this. The request seems to only delete attributes in error cases (`diagnoseAndRemoveAttr`). Is this call to `getParameterIndices()` only important for handling error cases, or is there a non-error case where it's also important?

Reply:
These calls to `getParameterIndices` are necessary to trigger `@differentiable` attribute type-checking for attributes in non-primary-files. Otherwise, attribute parameter indices and other fields may not be resolved.